docs/gettingstarted/k8s-microk8s.md

```diff
@@ -45,82 +37,86 @@ microk8s enable storage
 ```
-# Upgrade SC4S 
+
+# Update or upgrade SC4S 
+Whenever the image is upgraded or when changes are made to the `values.yaml` file should be applied, run the command:
 
 ```bash
 microk8s helm3 upgrade sc4s splunk-connect-for-syslog/splunk-connect-for-syslog -f values.yaml
 ```
 
-# Setup for HA with multiple nodes
-
-See https://microk8s.io/docs/high-availability
+# Install and configure SC4S for High Availability (HA)
 
-Note: Three identically-sized nodes are required for HA
+Three identically-sized nodes are required for HA. See [your Microk8s documentation](https://microk8s.io) for more information.
 
+1. Update the configuration file:
 ```yaml
 --8<---- "docs/resources/k8s/values_ha.yaml"
 ```
 
-Upgrade sc4s to apply the new config
+2. Upgrade SC4S to apply the new configuration:
+```bash
+microk8s helm3 upgrade sc4s splunk-connect-for-syslog/splunk-connect-for-syslog -f values.yaml
+```
 
-# Advanced Configuration
+# Configure your SC4S instances through `values.yaml`
 
-Using helm based deployment precludes direct configuration of environment variables and 
-context files but most configuration can be set via the values.yaml
+With helm-based deployment you cannot configure environment variables and 
+context files directly. Instead, use the `values.yaml` file to update your configuration, for example:
 
 ```yaml
 --8<---- "docs/resources/k8s/values_adv.yaml"
 
 ```
 
-`config_files` and `context_files` are variables used to specify configuration and context files that need to be passed to the splunk-connect-for-syslog.
+Use the `config_files` and `context_files` variables to specify configuration and context files that are passed to SC4S.
 
-`config_files`: This variable contains a dictionary that maps the name of the configuration file to its content in the form of a YAML block scalar.
-`context_file`: This variable contains a dictionary that maps the name of the context files to its content in the form of a YAML block scalar. The context file named splunk_metadata.csv and host.csv are being passed with the `values.yaml`
+- `config_files`: This variable contains a dictionary that maps the name of the configuration file to its content in the form of a YAML block scalar.
+- `context_file`: This variable contains a dictionary that maps the name of the context files to its content in the form of a YAML block scalar. The context files `splunk_metadata.csv` and `host.csv` are passed with `values.yaml`:
 ```yaml
 --8<---- "docs/resources/k8s/values_adv_config_file.yaml"
 ```
 
-# Resource Management
+# Manage resources
 
-Generally two instances will be provisioned per node adjust requests and limits to
-allow each instance to use about 40% of each node presuming no other workload is present
+Provision two instances per node. Adjust requests and limits to allow each instance to use about 40% of each node, presuming no other workload is present.
```

Comment:
What we mean here is that by default the user should expect two instances per node so they should assign about 40% of resource usage to each instance on every node, unless there is other workload. So maybe sth like:

```suggestion
You should expect two instances per node by default, so adjust requests and limits to allow each instance to use about 40% of each node, presuming no other workload is present.
```

Reply:
How about this: "You should expect your system to require two instances per node by default. Adjust requests and limits to allow each instance to use about 40% of each node, presuming no other workload is present. "

Reply:
great